usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -112,11 +115,15 @@ def test_cluster_create_valid(
         "present in cluster list.".format(integration_id))
 
     pytest.check(
-        len(imported_clusters["nodes"]) == len(nodes),
+        len(imported_clusters[0]["nodes"]) == len(nodes),
         "In cluster should be the same amount of hosts"
         "(is {}) as is in API call for cluster creation."
         "(is {})".format(len(imported_clusters["nodes"]), len(nodes)))
 
+    pytest.check(
+        [set(node_ids) == set(imported_clusters[0]["nodes"].keys())],
+        "There should be imported these nodes: {}".format(node_ids))
```

Comment:
Few problems here:
* Such check fill evaluate to True every time (no matter if the number of nodes matches or not): it's a list with a boolean value: both `bool([True])` and `bool([False])` would evaluate to `True` ...
* When this fails, would be the missing or extraneous node clearly visible from the logs? My guess it that not. (If this were normal pytest assert, then the answer would be yes, but we are using pytest.check() which doesn't provide all those magic features ...